packages/grafana-toolkit/src/plugins/env.ts

```diff
@@ -97,26 +97,22 @@ export const writeJobStats = (startTime: number, workDir: string) => {
   });
 };
 
+// https://circleci.com/api/v1.1/project/github/NatelEnergy/grafana-discrete-panel/latest/artifacts
 export async function getCircleDownloadBaseURL(): Promise<string | undefined> {
   try {
     const axios = require('axios');
-    const buildNumber = getBuildNumber();
     const repo = process.env.CIRCLE_PROJECT_REPONAME;
     const user = process.env.CIRCLE_PROJECT_USERNAME;
     let url = `https://circleci.com/api/v1.1/project/github/${user}/${repo}/latest/artifacts`;
     const rsp = await axios.get(url);
     for (const s of rsp.data) {
```

Comment:
I realize this is not part of the review, but could we have some stronger typing here? I makes things so much more readable.